requirements.txt

```diff
@@ -1,3 +1,3 @@
-discord.py
-requests
```

Comment:
This shouldn't be removed from the requirements file. Not all Python installs come with the requests module installed.